nextcloudappstore/core/user/views.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib import messages
+from django.shortcuts import render
+from django.utils.translation import ugettext as _
+
+import registration.backends.simple.views 
+
+from nextcloudappstore.core.user.forms import RegistrationFormRecaptcha
+
+
+class RegistrationView(registration.backends.simple.views.RegistrationView):
```

Comment:
Yes, I though about that as well, but...
`from registration.backends.simple.views.RegistrationView import RegistrationView`
doesn't work since I want our view to also be called RegistrationView. I could do 
`from registration.backends.simple.views.RegistrationView import RegistrationView as ....`
but I had no idea what to import it as so I opted for the abosulte path. :)

Do you have a suggestion?